### PR TITLE
Properly manage keys from initial data.

### LINKED ISSUE #6: Possible bug instantiating chest with data
Maybe I'm not using Chest right
```python
>>> from chest import Chest
>>> c = Chest({'k': 6})
>>> assert 'k' in c
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```

### PR DESCRIPTION
**Pull request by cowlicks**
Closes #6

### CODE REVIEW

File: chest/core.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def __init__(self, data=None, path=None, available_memory=None,
         # In memory storage
         self.inmem = data or dict()
         # A set of keys held both in memory or on disk
-        self._keys = {}
+        self._keys = set(data) if data is not None else {}
```
Comment (mrocklin) [inline chest/core.py]: Note that `{}` is an empty `dict`, not an empty `set`.

**Comment by cowlicks** [comment]
The exception is in coverage...?

**Comment by mrocklin** [comment]
Feel free to remove 3.2 from the travis.yml.  Can I ask for a rebase of the "put keys", "test keys" and "pep8" commits and (if you don't mind me being a bit anal) use of the full word "you" in the gitignore commit.

**Comment by cowlicks** [comment]
@mrocklin sure, here u go :)

**Comment by mrocklin** [comment]
Thanks!  Merged.

**Pull request merged**